lighthouse-core/report/html/renderer/report-ui-features.js

```diff
@@ -380,6 +402,29 @@ class ReportUIFeatures {
     this._document.body.removeChild(a);
     setTimeout(_ => URL.revokeObjectURL(href), 500);
   }
+
+  _handleStickyHeader() {
```

Comment:
maybe rename to make it clear this is for adjusting the sticky header on scroll events?

Reply:
It handles sticky header visibility, and the highlighting. It also fires on scroll and resize.

`_handleStickyHeaderVisibilityAndHighlighter`? :)

Reply:
haha, I was thinking more like `_updateStickyHeaderOnScroll` or whatever :)

Arguably (it's a stretch :) resize is a special case of scrolling for this element, resizing is uncommon compared to scrolling, and the main thing is the name can communicate something about what it's up to and when

Reply:
sold, to `_updateStickyHeaderOnScroll`